ui/ui-components/pages/app/[id]/edit.tsx

```diff
@@ -244,6 +247,7 @@ export default function Page(props) {
                           <a
                             target="_blank"
                             href="https://www.grouparoo.com/docs/support/secrets"
+                            rel="noreferrer"
```

Comment:
Linter will spot this when we add it to these apps. Just getting a little ahead. :)